frontend/host/src/Site.tsx

```diff
@@ -92,7 +92,7 @@ export const Site: FC = () => {
                   />
                 }
               />
-              <Route path={'/details/:code'} element={<RedirectDetails />} />
+              <Route path={'/detail/:code'} element={<RedirectDetails />} />
```

Comment:
Was this deliberate? I can't remember what the old link was like...

Reply:
Oh whoops didn't realise that made it in here... when we deployed the production version i realised the old links were /detail/:code, but we'd set up the redirect off /details... will do via separate PR for clarity 😎